packages/dd-trace/src/proxy.js

```diff
@@ -123,6 +123,15 @@ class Tracer extends BaseTracer {
   bindEmitter () {
     this._deprecate('bindEmitter')
   }
+
+  injectRumData () {
```

Comment:
I know I'm late to the party, but this method name gives the impression that it does a mutation and doesn't return a value. As a user I would expect to call it and that injection would work without grabbing any return value. Can we rename this one before it's released?